renderer/canvas.py

```diff
@@ -679,13 +684,44 @@ def _extract_stations_to_furniture(self, abstract_tile_matrix):
         Returns:
             asbtract_tile_matrix (List[List[String]]): matrix with furniture tiles added
         """
+
+        stations = []
         abstract_tile_matrix = [[abstract_tile_matrix[i][j] for j in range(len(abstract_tile_matrix[i]))]for i in range(len(abstract_tile_matrix))]
         for i, row in enumerate(self.layout):
             for j, col in enumerate(row):
                 if col is not None:
                     asset_info = self._choose_station_asset(col)
                     if asset_info["type"] == "tile":
                         abstract_tile_matrix[i][j] = asset_info["name"]
+                    else:
+                        name, _ = trim_item_ID(col)
+                        stations.append((i,j, name))
+        
+        directions = [(1,0), (0,1), (-1,0), (0, -1)]
+
+        row = len(abstract_tile_matrix)
+        col = len(abstract_tile_matrix[0])
+        for (x,y, station_name) in stations:
+            tables = 0
+            counters = 0
+            underneath = self.config["station"]["entities"][station_name]["constants"].get("underneath", None)
+
+            if underneath is None:
```

Comment:
shouldn't this if statement be `if underneath is not None` since the code should assign the appropriate table in the case that one isn't specified?

Reply:
I read in the PR that `underneath` is only used in the case that the station is an island; good information - include it in the helper function docstring.

Reply:
I believe it should be `if underneath is None:`, because this is just checking whether there is a corresponding constant in the file for the station. If there is no corresponding constants, then this just means there should not be a platform underneath the station. For example, stove does not have a `underneath` constant, meaning there should not be a platform underneath